packages/next/src/lib/verify-partytown-setup.ts

```diff
@@ -50,8 +49,7 @@ async function copyPartytownStaticFiles(
   )
 
   if (hasPartytownLibDir) {
-    await recursiveDelete(partytownLibDir)
-    await promises.rmdir(partytownLibDir)
+    await promises.rm(partytownLibDir, { recursive: true, force: true })
```

Comment:
Similarly here, @housseindjirdeh was there a reason for both `recursiveDelete()` and `promises.rmdir()`? It was added in:

- https://github.com/vercel/next.js/pull/34244

And does the new `promises.rmdir()` solve this use case?